### PR TITLE
Task3 6

### PR DESCRIPTION
**Pull request by KatGaertner**
Implemented Bootstrap and some styling.

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+  font-family: "Comfortaa", cursive;
+}
+
+// .card-title {
+//   display: -webkit-box;
+//   -webkit-line-clamp: 3;
+//   -webkit-box-orient: vertical;
+//   overflow: hidden;
+// }
+
```
Comment (Stanley-Okwii) [inline src/index.scss]: It's good practice to remove any unused commented code.

File: src/components/signup-view/signup-view.jsx [hunk 2]
```diff
@@ -44,59 +49,99 @@ export const SignupView = () => {
...
+                } else {
+                  e.target.setCustomValidity("");
+                }
+              }}
+            />
+          </Col>
+          <Col sm={{ span: rightColumnWidth, offset: leftColumnWidth }}>
+            <Form.Text id="usernameText">
+              Usernames must have at least 5 characters and use only
+              alphanumeric characters.{" "}
```
Comment (Stanley-Okwii) [inline src/components/signup-view/signup-view.jsx:88]: ❓What the purpose of this code: `{" "}` here?

File: src/components/signup-view/signup-view.jsx [hunk 3]
```diff
@@ -44,59 +49,99 @@ export const SignupView = () => {
...
-          value={password}
-          onChange={(e) => setPassword(e.target.value)}
-          required
-          minLength={8}
-        />
-      </label>
-      <button type="submit">Submit</button>
-    </form>
+    <div class="p-4 rounded-4 bg-body">
+      <h1 class="text-center">Sign up</h1>
```
Comment (Stanley-Okwii) [inline src/components/signup-view/signup-view.jsx]: Could you please replace all instances of `class` with `ClassName`.  React yells about this in console.
Screenshot 2023-06-26 at 10 23 53 PM

**Review by Stanley-Okwii — APPROVED** [review]
Great work @KatGaertner . I have left a couple of comments to help you polish up your work.

File: src/components/signup-view/signup-view.jsx (hunk 3 above)
Comment (KatGaertner) [reply src/components/signup-view/signup-view.jsx]: Sorry! I had replaced them for a later version, but I forgot to double-check here.

File: src/components/signup-view/signup-view.jsx (hunk 2 above)
Comment (KatGaertner) [reply src/components/signup-view/signup-view.jsx:88]: I think this is something Prettier did when the whitespace was needed. It's not needed anymore so I removed it.

**Pull request merged**